rules/DowngradePhp80/Rector/Class_/DowngradeAttributeToAnnotationRector.php

```diff
@@ -105,9 +106,14 @@ public function refactor(Node $node): ?Node
         $this->isDowngraded = false;
 
         $phpDocInfo = $this->phpDocInfoFactory->createFromNodeOrEmpty($node);
+        $requireReprint = false;
         foreach ($node->attrGroups as $attrGroup) {
             foreach ($attrGroup->attrs as $key => $attribute) {
                 if ($this->shouldSkipAttribute($attribute)) {
+                    // avoid error on same line
+                    // as attribute ->getStartLine() always equal with node ->getStartLine()
+                    // can't validate it, so enforce to reprint later
+                    $requireReprint = true;
```

Comment:
this seems cause downgrade error, I will revert this part

https://github.com/rectorphp/rector-src/actions/runs/11986059063/job/33418819341#step:14:67

Reply:
https://github.com/rectorphp/rector-downgrade-php/pull/249